zzzz_modular_occulus/code/modules/research/nodes/combat.dm

```diff
@@ -63,3 +63,22 @@
 						   /datum/design/research/item/weapon/greyclaw/greyclawmag/rubber,
 						   )
 
+// Railguns
+
+/datum/technology/railguns
+	name = "Hyper Velocity Weaponry"
+	desc = "Prototype weaponry that uses electromagnetic propulsion to send projectiles."
+	tech_type = RESEARCH_COMBAT
+
+	x = 0.6
+	y = 0.7
+	icon = "railgun"
+
+	required_technologies = list(/datum/technology/temp)
+	required_tech_levels = list(RESEARCH_ENGINEERING = 5, RESEARCH_POWERSTORAGE = 4)
+	cost = 6000
+
+	unlocks_designs = list(/datum/design/research/item/weapon/railgun,
+							/datum/design/research/item/weapon/gauss
```

Comment:
Given the difference in power, it may be a good idea to shuffle Gauss into a separate tech. That being said, definitely not necessary.